docs/HowToFAQ/FAQSplitHere.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+---
+
+## Question
+
+How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+
+## Answer
+
+Analysis of a method for the purpose of verification identifies a large number of assertions that must be proved.
+These assertions are grouped into batches.
+
+One default way of operating is to combine all assertions into one batch, make one verification condition out of it,
```

Comment:
"make one verification condition out of it" consider replacing this with "leading to a single [prover/solver] run"

Reply:
OK